open-company/open-company-web
# Emojione picker

Card: https://trello.com/c/w7tfNFwY

@belucid this needs a one time migration of the db data to change the actual emojis to unicode.

To test:
- try to use the picker in:
- FoCE: headline, snippet
- fullscreen edit: headline snippet (no body)
- SU share: email and slack note
- check that the data sent to the API has only unicode chars and not html img tags
- check that the data coming from API have emojis represented in unicode and that they are translated to html img tags when rendered inside React components
